tests/native_tests/core.hy

```diff
@@ -710,7 +710,7 @@ result['y in globals'] = 'y' in globals()")
             f"Help on function {(mangle '<-mangle->)} in module "))
   (assert (in "a fancy docstring" out))
   (assert (empty? err))
-  #doc @
+  (doc "#@")
```

Comment:
We should just remove this line, as it causes intermittent test failures. Builtin macros are not always loaded at runtime, and this is likely going remain intentional.

Reply:
I put another `defmacro` in the test instead.